src/sinks/blackhole/config.rs

```diff
@@ -19,13 +23,17 @@ pub struct BlackholeConfig {
     /// The number of seconds between reporting a summary of activity.
```

Comment:
@spencergilbert , since it's similar to issues you flagged on my PR :)

, should this be : 

```suggestion
    /// The interval between reporting a summary of activity.
```

?

Reply:
Slightly different :P but yeah, if we have a `type_unit` of "foobar" we don't need to specify it in the description as well